### DISCUSSION TITLE
How to convert prediction score into prediction label in deepgcn model?

### DISCUSSION THREAD

**Discussion by EimhinLiu** [pyg-team/pytorch_geometric#4789] [Q&A]
Hi PyG Community,

Recently, I was running an example from the PyG examples, ogbn_proteins_deepgcn(https://github.com/pyg-team/pytorch_geometric/blob/master/examples/ogbn_proteins_deepgcn.py). I found that when evaluating 'rocauc', it used the prediction score of the model, that is, 'out[mask]'. But now I want to convert this prediction score into prediction classification label. That is to say, change the value of 'y_pred' to be the same as that of 'y_true'. Is there any way to implement it?

**Comment by rusty1s** [comment]
Yes, you can just do that via a threshold:
```python
out = model(...)
pred = out > 0.0
```

**Reply by EimhinLiu** [reply]
Thank you for that, it worked!
